<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DAChartSettingWidget</class>
 <widget class="QWidget" name="DAChartSettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>844</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chart Setting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="toolButtonFigure">
       <property name="text">
        <string>Figure</string>
       </property>
       <property name="icon">
        <iconset resource="icon.qrc">
         <normaloff>:/DAGui/icon/figure-setting.svg</normaloff>:/DAGui/icon/figure-setting.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroupType</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonChart">
       <property name="text">
        <string>Chart</string>
       </property>
       <property name="icon">
        <iconset resource="icon.qrc">
         <normaloff>:/DAGui/icon/plot-setting.svg</normaloff>:/DAGui/icon/plot-setting.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroupType</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonItem">
       <property name="text">
        <string>item</string>
       </property>
       <property name="icon">
        <iconset resource="icon.qrc">
         <normaloff>:/DAGui/icon/plot-item-setting.svg</normaloff>:/DAGui/icon/plot-item-setting.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroupType</string>
       </attribute>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxSelectChart"/>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxSelectItem"/>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="pageFigureSet"/>
     <widget class="DA::DAChartPlotSettingWidget" name="widgetChartSetting">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
      </layout>
     </widget>
     <widget class="DA::DAChartCommonItemsSettingWidget" name="widgetItemSetting"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DA::DAChartCommonItemsSettingWidget</class>
   <extends>QWidget</extends>
   <header>DAChartCommonItemsSettingWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DA::DAChartPlotSettingWidget</class>
   <extends>QWidget</extends>
   <header>DAChartPlotSettingWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>toolButtonItem</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBoxSelectItem</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonChart</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBoxSelectChart</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonChart</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBoxSelectItem</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonItem</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBoxSelectChart</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonFigure</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBoxSelectChart</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>23</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>26</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonFigure</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBoxSelectItem</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>15</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroupType"/>
 </buttongroups>
</ui>
